framework/encode/dx12_state_tracker.cpp

```diff
@@ -618,17 +741,39 @@ void Dx12StateTracker::TrackBuildRaytracingAccelerationStructure(
             build_info.inputs.pGeometryDescs  = nullptr;
             build_info.inputs.ppGeometryDescs = nullptr;
         }
+        else if (desc->Inputs.Type == D3D12_RAYTRACING_ACCELERATION_STRUCTURE_TYPE_TOP_LEVEL)
+        {
+            // TEMP: Force all TLAS builds to take this code path for testing.
+            // if (desc->Inputs.DescsLayout == D3D12_ELEMENTS_LAYOUT_ARRAY_OF_POINTERS)
```

Comment:
This conditional is temporarily commented out for testing. Build TLAS with D3D12_ELEMENTS_LAYOUT_ARRAY_OF_POINTERS isn't used heavily, so this forces all Build TLAS calls to take the new code path. Once tests pass and before merging to `dev` this conditional will be restored.